app/services/risk_assessment/high_risk_assessment.rb

```diff
@@ -29,7 +29,7 @@ def enhancement_applied?
     end
 
     def uplift_applied?
-      @items[:work_items].work_item_forms.filter_map(&:uplift).any?
+      @items[:work_items].work_item_forms.filter_map(&:uplift).any?(&:positive?)
```

Comment:
I think this would work better as:

```ruby
 @items[:work_items].work_item_forms.any? { _1.uplift.positive? }
 ```
 
 That saves us iterating through the work array twice.

Reply:
Thanks @patrick-laa , I had to change a bit from what you suggested. as uplift can be nil, and I was getting 'positive?' doesn't have a method nil in my rspec. to fix it I have used lonely operator '&.positive?' so 'positive?' is only called if '_1.uplift' is not nil.

`@items[:work_items].work_item_forms.any? { _1.uplift&.positive? }`